test/Microsoft.ML.Tests/Scenarios/Api/Estimators/Metacomponents.cs

```diff
@@ -36,5 +35,105 @@ public void New_Metacomponents()
                 var model = pipeline.Fit(data);
             }
         }
+
+        /// <summary>
+        /// OVA with calibrator
+        /// </summary>
+        [Fact]
+        public void New_OVAWithCalibrator()
```

Comment:
>New_OVAWithCalibrator [](start = 20, length = 21)

move these to a different location: we don't need to expand the set of estimator tests